pytrip/utils/slicedos.py

```diff
@@ -0,0 +1,211 @@
+from pytrip import dos
+from pytrip import ctx
+import argparse
+import logging
+from numpy import arange, meshgrid, ma
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import os
+import sys
+
+
+def check_compatible(a, b):
+    """ Simple comparison of cubes. if X,Y,Z dims are the same, and
+    pixel sizes as well, then they are compatible. (Duck typed)
+    """
+    eps = 1e-5
+
+    if a.dimx != b.dimx:
+        logging.error("DIMX does not match: "+str(a.dimx)+" "+str(b.dimx))
+        raise Exception("Cubes don't match, check dimx in header.")
+
+    if a.dimy != b.dimy:
+        logging.error("DIMY does not match: "+str(a.dimy)+" "+str(b.dimy))
+        raise Exception("Cubes don't match, check dimy in header.")
+
+    if a.dimz != b.dimz:
+        logging.error("DIMZ does not match: "+str(a.dimz)+" "+str(b.dimz))
+        raise Exception("Cubes don't match, check dimz in header.")
+
+    if (a.pixel_size - b.pixel_size) > eps:
+        logging.error("Pixel size does not match: "+str(a.pixel_size)+" "+str(b.pixel_size))
+        raise Exception("Cubes don't match, check pixel_size in header.")
+
+    if a.slice_dimension != b.slice_dimension:
+        logging.error("Slice dimension does not match: "+str(a.slice_dimension)+" "+str(b.slice_dimension))
+        raise Exception("Cubes don't match, check slice_dimension in header.")
+    return True
+
+
+def main():
```

Comment:
To have working tests you need to have `main` with arguments, see:
https://github.com/pytrip/pytrip/blob/master/pytrip/utils/trip2dicom.py#L9
`def main(args=sys.argv[1:]):`
and pass these aguments to `argparse.ArgumentParser`

https://github.com/DataMedSci/pymchelper/blob/master/pymchelper/run.py#L52
` parsed_args = parser.parse_args(args)`

Having args enable you to use them in tests, see:
https://github.com/pytrip/pytrip/blob/master/tests/test_converters.py#L21
` pytrip.utils.trip2dicom.main([self.cube000, tmpdir])`